src/stage/mouse-observer.ts

```diff
@@ -228,15 +230,18 @@ class MouseObserver {
    * @param  {Event} event - mouse event
    * @return {undefined}
    */
-  _onMousewheel(event: WheelEvent & { wheelDelta?: number, wheelDeltaY?: number }) {
+  _onMousewheel(event: Optional<WheelEvent, 'detail'> & { wheelDelta?: number, wheelDeltaY?: number }) {
     if (event.target !== this.domElement || !this.handleScroll) {
       return
     }
     event.preventDefault()
-    this._setKeys(event)
+    this._setKeys({...{detail: 0}, ...event})
```

Comment:
Argh, this was a tricky one: The spread operator (and the Object.assign call that it gets transpiled into) both don't copy the prototype, so things like shiftKey modifiers aren't captured. I fiddled a bit with types but given we're dealing with old browsers and suchlike I think it's fair to put `event as any` here?

Reply:
I'm fine with that.